proxy/src/main/java/com/velocitypowered/proxy/command/VelocityCommandManager.java

```diff
@@ -242,7 +241,7 @@ public CompletableFuture<Boolean> executeAsync(final CommandSource source, final
     Preconditions.checkNotNull(cmdLine, "cmdLine");
 
     return callCommandEvent(source, cmdLine).thenApplyAsync(event -> {
-      CommandResult commandResult = event.getResult();
+      CommandExecuteEvent.CommandResult commandResult = event.getResult();
```

Comment:
This is an unfortunate name clash, but nothing we can do yet. We could merge both enums in the next major version.